### ISSUE TITLE
Use Mapillary vector tiles in order to get to show all images

### ISSUE THREAD

**Issue by peterneubauer** [openstreetmap/iD#3152]
In the current implementation of creating a layer for every Mapillary image, we have a limit of 1000 images per request (tile) to the mapillary API, resulting in empty spots or images that are not shown, like in https://www.openstreetmap.org/node/244646675#map=16/-41.4369/147.1385 where there is a lot of coverage.

It would be better to use the Mapillary vector tiles, like e.g. https://d2munx5tg0hw47.cloudfront.net/tiles/14/8786/5127.mapbox that gives sequences and image location/compass angle data.

I don't know exactly how to integrate vector tiles with the D3 transforming scheme, so help from someone more skilled than me would be hugely appreciated!

**Comment by bhousel** [comment]
Yeah, this is a good next step.  We are [currently](https://github.com/openstreetmap/iD/blob/master/js/id/services/mapillary.js#L9-L10) fetching up to 10 pages of data per z14 tile from the paginated API.   

Before f2cde0b we fetched *all* the data, but this ended up being too much around Knoxville, TN 😄   There are probably a few million images there.  I've also noticed Mapillary JS viewer timeouts around there, maybe it is not able to efficiently cache nearby images.

But yeah, lets switch over to the vector tiles!
